pkg/api/blocks/v1.go

```diff
@@ -32,8 +39,27 @@ type BlocksInfo struct {
 	Err         error           `json:"err"`
 }
 
+type ActionType int32
+
+const (
+	Deletion ActionType = iota
+	NoCompaction
+	Unknown
+)
+
+func parse(s string) ActionType {
+	switch s {
+	case "DELETION":
+		return Deletion
+	case "NO COMPACTION":
```

Comment:
```suggestion
	case "NO_COMPACTION":
```

Just a suggestion. Whitespace doesn't always play nice in bash scripts etc. so maybe it's better to keep these constants a single word.